projects/netgrif-components/src/lib/data-fields/number-field/number-currency-field/number-currency-field.component.html

```diff
@@ -9,8 +9,8 @@
            [formControl]="formControlRef"
            [placeholder]="dataField.placeholder"
            [required]="dataField.behavior.required"
-           (focusout)="onFocusOut($event)"
-           (focusin)="onFocusIn()">
+           (focusout)="dataField.unsetFocus(); onFocusOut($event)"
+           (focusin)="dataField.setFocus(); onFocusIn()">
```

Comment:
if there are `onFocusOut` and `onFocusIn` methods already, why not put the calls inside of these methods in the `.ts` file?